hotel-booking/src/main/java/nostra/cosa/hotelbooking/service/service/impl/AccommodationServiceImpl.java

```
@@ -33,7 +46,12 @@ public AccommodationDTO create(AccommodationDTO newAccommodation) {
   }
 
   @Override
-  public void delete(Long id) {
-
+  public Boolean delete(Long id) {
+    try {
+      accommodationRepository.deleteById(id);
+      return true;
+    } catch (Exception e) { // TODO: egyenlőre nem találtam konkrétabbat ha később lenne itt hagyom a todot
+      return false;
```

doksiban:

deleteById

Deletes the entity with the given id.
If the entity is not found in the persistence store it is silently ignored.

Parameters:
id - must not be null.
Throws:
[IllegalArgumentException] - in case the given id is null

Tehát akkor az IllegalArgumentException-t el lehetne kapni.